uint8 START_CALIB = 1
uint8 STOP_CALIB = 0

uint8 status

float32 xMax
float32 xMin

float32 yMax
float32 yMin

float32 zMax
float32 zMin

---
bool ack
